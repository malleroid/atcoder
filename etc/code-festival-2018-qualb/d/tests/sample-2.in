2 3 10
1 3
3 2
9 5
