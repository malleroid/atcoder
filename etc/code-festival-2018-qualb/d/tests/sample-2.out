4.1600000000
